docs/docs-w-demo-ribbon.less

```diff
@@ -0,0 +1,147 @@
+/* ==================================================== *\
+ * Styles below are NOT distributed with HelixUI assets *
+\* ==================================================== */
+@import (reference) 'vars';
+// import grid mixins from toolkit
+@import (reference) 'modules/grid';
+
+@demo-container-border: 1px dashed @gray-600;
+@demo-fill-color: fade(@orange-500, 25%);
+@demo-padding-color: @magenta-100;
+
+[v-cloak] {
+  opacity: 0;
+}
+
+.homepage {
+  background-image: linear-gradient(0deg, @cyan-100, @gray-0 20%);
+}
+
+.brand-logo {
+  align-items: center;
+  display: flex;
+  height: 3.5rem;
+  justify-content: center;
+  width: @app-nav-width;
+
+  img {
+    height: 2rem;
+  }
+}
+
+.demo {
+  background-color: @gray-0;
+  border: 1px solid #ccc;
+  overflow: hidden;
+  padding: 1.25rem;
+  position: relative;
+
+  > pre {
+    margin-top: 1.25rem;
+  }
+
+  &::after {
+    background-color: #ccc; //@gray-500;
+    color: @gray-900;
+    content: 'DEMO';
+    font-size: 0.5rem;
+    font-weight: bold;
+    letter-spacing: 1px;
+    line-height: 1;
+    padding: 4px;
+    position: absolute;
+    right: -20px;
+    text-align: center;
+    top: 6px;
+    transform: rotate(45deg);
+    width: 4rem;
+  }
+}
+
+// Table Of Contents
+.article-toc {
+  display: block;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 1rem;
+
+  > a {
+    display: block;
+    font-weight: 500;
+    margin-bottom: 0.5rem;
+    text-decoration: none;
+
+    &:hover {
+      //text-decoration: underline;
```

Comment:
Since the only property is commented out, should this selector be removed?

Reply:
This file shouldn't have been added. Removing.